shell/platform/darwin/ios/ios_surface_gl.mm

```diff
@@ -84,7 +84,7 @@
 
 // |GPUSurfaceGLDelegate|
 ExternalViewEmbedder* IOSSurfaceGL::GetExternalViewEmbedder() {
-  return this;
+  return GetSurfaceExternalViewEmbedder().get();
```

Comment:
Can `GetExternalViewEmbedder` now move to `GPUSurfaceDelegate`?

Reply:
I plan to delete `GetExternalViewEmbedder` completely from the surface interfaces in follow up commits.